backend/service/auditsink/slack/slack.go

```diff
@@ -104,3 +124,96 @@ func formatText(username string, event *auditv1.RequestEvent) string {
 
 	return messageText
 }
+
+// FormatCustomText parses out the audit event metdata request for the custom slack message text
+func FormatCustomText(message string, event *auditv1.RequestEvent) (string, error) {
+	tmpl, err := template.New("customText").Funcs(funcMap).Parse(message)
+	if err != nil {
+		return "", err
+	}
+
+	data, err := getAuditTemplateData(event)
+	if err != nil {
+		return "", err
+	}
+
+	var buf bytes.Buffer
+	if err := tmpl.Execute(&buf, data); err != nil {
+		return "", err
+	}
+
+	// When a value is nil, the Go Template returns "<no value>" in its place. Replacing this with null instead.
+	// We hit this scenario with https://github.com/lyft/clutch/blob/main/api/k8s/v1/k8s.proto#L860
+	sanitized := strings.ReplaceAll(buf.String(), "<no value>", "null")
```

Comment:
wrote a unit test here https://github.com/lyft/clutch/pull/1093/files#diff-17fcfcb6bbd591d6dcd0e04f6a9ceb6bb8b2d0cba8ee7ecf84f1cf907ae73dccR134

Reply:
should we use `None` here as well?
```suggestion
	sanitized := strings.ReplaceAll(buf.String(), "<no value>", "None")
```